package.json

```diff
@@ -18,6 +18,9 @@
     "nightwatch": "0.9.6",
     "nightwatch-commands": "1.7.0"
   },
+  "engines": {
+      "node": "^4.0.0"
```

Comment:
That's a pretty good idea @jeremywiebe , but I believe it won't be enforced by node v6 / npm v3 so we'd only be covering the cases of node v0 -> v4.

That's better than just leaving it as advisory though, so I'll make that change.